x/bank/client/cli/tx_test.go

```diff
@@ -64,17 +64,6 @@ func (s *CLITestSuite) TestSendTxCmd() {
 			extraArgs,
 			true,
 		},
-		{
```

Comment:
I would add an extra check in the CLI for avoiding removing this (if the check is easy)